candidate-standard/abstract_tests/ATS_class_queries.adoc

```diff
@@ -46,7 +46,7 @@ include::collections/ATS_rc-op.adoc[]
 include::collections/ATS_rc-response.adoc[]
 
 ===== Instances {root}/collections/{collectionId}/instances
-
+// should these point to instances/ATS_rc-md... files?
```

Comment:
See #102 I'm not quite sure what files should be referenced here, but the include statements are currently failing.